server/cluster_leader.go

```diff
@@ -179,6 +179,7 @@ func (c *Cluster) sendPings() {
 
 		c.fo.activeNodes = activeNodes
 		c.rehash(activeNodes)
+		c.invalidateRemoteSubs(globals.sessionStore)
```

Comment:
It does not make a ton of sense to pass global var `globals.sessionStore` as parameter. Unless you want to get rid of globals completely. which is not going to be trivial.

Reply:
Done. Using the global now.

Reply:
I don't see the change

Reply:
Just did.